src/content_api/content_api_extract.py

```diff
@@ -159,37 +96,38 @@ def compute_chunks(start, end, chunk_size):
             range(start, end, chunk_size)]
 
 
-def merge_delete(directory):
+def merge_delete(directory, prefix):
     logging.info("Merging and deleting created subfiles...")
-    files_to_del = merge_dataframe(directory)
+    files_to_del = merge_dataframe(directory, prefix)
     for file in files_to_del:
         logging.debug("Deleting {}...".format(file))
         os.remove(file)
 
 
-def merge_dataframe(directory):
+def merge_dataframe(directory, prefix):
     all_files = []
     for keyword in ["json", "api_links"]:
-        filelist = sorted([os.path.join(directory, file) for file in os.listdir(directory) if keyword in file])
+        filelist = sorted(
+            [os.path.join(directory, file) for file in os.listdir(directory) if keyword in file and "pt" in file])
```

Comment:
we could potentially do a regex match here, using `_content_json` and `_content_api_links` and `pt` to be more sure of what we're selecting, but may just be overkill

Reply:
Actually forget that, I see you make and delete a folder for this stuff